financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/SharedPartnerAuth.kt

```diff
@@ -80,7 +76,8 @@ internal fun SharedPartnerAuth(
     onCancelClick: () -> Unit,
     onClickableTextClick: (String) -> Unit,
     onWebAuthFlowFinished: (WebAuthFlowState) -> Unit,
-    onViewEffectLaunched: () -> Unit
+    onViewEffectLaunched: () -> Unit,
+    inModal: Boolean
```

Comment:
TLDR: `inModal` will not render the spacer that fills all available space between the body and the footer, so that it displays properly within a bottom sheet.